openapi/openapi.json

```diff
@@ -958,6 +958,132 @@
         }
       ]
     },
+    "/bundles/bundle-request/{id-bundle-request}/creditor-institutions/{ci-tax-code}": {
```

Comment:
better like this `/bundles/creditor-institutions/{ci-tax-code}/requests/{bundle-request-id}`